sympy/sets/sets.py

```diff
@@ -381,6 +381,25 @@ def _contains(self, other):
         """
         raise NotImplementedError(f"{type(self).__name__}._contains")
 
+    def convert_to_interval(set):
+        if isinstance(set, Interval):
+            return set
+        elif isinstance(set, Union):
+            intervals = [i for i in set.args if isinstance(i, Interval)]
+            if intervals:
+                lower_bound = Min([i.args[0] for i in intervals])
+                upper_bound = Max([i.args[1] for i in intervals])
+                return Interval(lower_bound, upper_bound)
```

Comment:
This is not equivalent to the original set e.g. `Interval(1, 2) | Interval(5, 6)` would be turned into `Interval(1, 6)` but that is not equivalent because it also includes numbers in between 2 and 5.